CMakeLists.txt

```diff
@@ -1101,6 +1101,9 @@ if(NOT MSVC)
     append_cxx_flag_if_supported("-Wno-error=redundant-move" CMAKE_CXX_FLAGS)
   endif()
 else()
+  # Define export functions for AOTI.
+  add_compile_definitions(EXPORT_AOTI_FUNCTIONS)
```

Comment:
So looks like the problem is very simple, someone needs to add the same define to respective bzl file internally, otherwise it will not work

Reply:
For more context, I attempted to do that before the holiday in https://www.internalfb.com/diff/D67525962?dst_version_fbid=937729108340657&transaction_fbid=435395002838074, but I probably did it wrong or missed other places so there were still some build failures left [T211170671](https://www.internalfb.com/intern/tasks/?t=211170671).  I didn't have time to follow up on the issue at that time, so I did a revert instead.

Reply:
After some debugging, apparently there is a case when someone tries to load two instances of libtorch (one linked as shared library and another statically) into the same address space, and this causes change to the internal build system (that @huydhn proposed) to fail....

As AOTI is not used in internal Windows deployments, in short term I propose just disable it, but in the long run someone needs to modify buck to mark two targets incompatible with each other

Reply:
But I don't understand from PR description, why this define needs to be here

Reply:
@xuhancn why this change needs to be there? I.e. shouldn't it be a private define of say `torch_cpu`?